### ISSUE TITLE
[Tracking Issue] PyTorch Frontend Operator Support

### ISSUE THREAD

**Issue by AndreSlavescu** [hidet-org/hidet#132]

**Comment by AndreSlavescu** [comment]
## torch.nn.functional operators

### non-linear activations 

- [ ] elu
- [ ] relu
- [ ] relu6
- [ ] celu
- [ ] hardsigmoid
- [ ] sigmoid
- [ ] hardtanh
